TCAT/Utilities/Analytics.swift

```diff
@@ -218,3 +208,13 @@ struct WhatsNewCardDismissedPayload: Payload {
 
     let actionDescription: String
 }
+
+/// Log get errors
+struct GetErrorPayload: Payload {
+    static let eventName: String = "Get Error"
```

Comment:
Can we change the name of this to be `NetworkError` and the eventName to `Network Error`

Reply:
I didn't go with `NetworkErrorPayload` bc not all of the errors were actually network error. For example, in `RouteOptionsVC` there's an error in getting routes for when place coordinates are not valid.

Reply:
Hmm i see. Lets still change it to `NetworkErrorPayload` because even tho some of the errors aren't specifically using the result of a network request, they still have to do with the information being passed around during network requests.